asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js

```diff
@@ -0,0 +1,213 @@
+    ProjectAPI.fetchExportProject,
+    {
+      enabled: file === "project" && exporting,
+      refetchOnWindowFocus: false,
+      onSettled: () => setExporting(false),
+    }
+  );
+
+  const selectedQuery = () => {
+    if (file === "dataset") {
+      return [exportDatasetQuery, "fetchExportDataset"];
+    }
+    if (file === "project") {
+      return [exportProjectQuery, "fetchExportProject"];
+    }
+  };
+
+  const handleFile = (event) => {
+    setFile(event.target.value);
+    if (event.target.value === "project") {
+      setFileFormat("asreview");
+    } else {
+      setFileFormat("");
+    }
+  };
+
+  const handleFileFormat = (event) => {
+    setFileFormat(event.target.value);
+  };
+
+  const onClickExport = () => {
+    setExporting(true);
+  };
+
+  const disableExportButton = () => {
+    return !file || !fileFormat || exporting;
+  };
+
+  return (
+    <Root aria-label="export page">
+      {selectedQuery() && (
+        <SnackbarErrorHandler
+          severity="error"
+          open={selectedQuery()[0].isError}
+          message={`${selectedQuery()[0].error?.message} Please try again.`}
+          queryKey={selectedQuery()[1]}
+        />
+      )}
+      {selectedQuery() && (
+        <SnackbarErrorHandler
+          severity="success"
+          open={selectedQuery()[0].isSuccess}
+          message="Successfully exported the file."
+          queryKey={selectedQuery()[1]}
+        />
+      )}
+      <Fade in>
+        <Box className={classes.root}>
+          <Stack direction="column" spacing={5} className={classes.select}>
+            <FormControl sx={{ height: 86 }}>
+              {!file && (
+                <InputLabel id="file-select-label" shrink={false}>
+                  Select file
+                </InputLabel>
+              )}
+              <Select
+                labelId="file-select-label"
+                id="file-select"
+                value={file}
+                onChange={handleFile}
+              >
+                <MenuItem
+                  value="dataset"
+                  disabled={!props.enableExportDataset}
+                  divider
+                >
+                  <Box>
+                    <Typography variant="subtitle1">Dataset</Typography>
+                    <Typography
+                      variant="body2"
+                      gutterBottom
+                      sx={{ color: "text.secondary" }}
+                    >
+                      With relevant/irrelevant labels
+                    </Typography>
+                  </Box>
+                </MenuItem>
+                <MenuItem value="project">
+                  <Box>
+                    <Typography variant="subtitle1">Project</Typography>
+                    <Typography
+                      variant="body2"
+                      gutterBottom
+                      sx={{ color: "text.secondary" }}
+                    >
+                      Including data and model configuration
```

Comment:
@SagevdBrand @Rensvandeschoot Can you think about the wording?

Reply:
Including all information on model settings?